Makefile

```
@@ -0,0 +1,4 @@
+install:
+	docker-compose build
+dev:
+	docker-compose up -d
```

De acordo com minha sugestão sobre os comandos Make:
```suggestion
build:
	docker-compose build
up:
	docker-compose up -d
```